scispacy/version.py

```diff
@@ -1,6 +1,6 @@
 _MAJOR = "0"
 _MINOR = "1"
-_REVISION = "0-unreleased"
```

Comment:
Revert this, this is only for actual releases

Reply:
The idea is that when we do a release, we upgrade from "0.1.0-unreleased" to "0.1.0", do the release, then push a commit which changes to "0.1.1-unreleased", so that our release has a fixed, specified version and if anyone wants to install from master, their version will be "0.1.1-unreleased" so that they know that it wasn't an officially packaged one.